site/tutorials/keptn-multistage-qualitygates-09.md

```diff
@@ -128,31 +128,38 @@ In the `shipyard.yaml` shown above, we define two stages called *hardening* and
 ## Create a service
 Duration: 2:00
 
-After creating the project, we can continue by onboarding the *helloserver* as a service to your project using the `keptn create service` and `keptn add-resource` commands. You need to pass the project where you want to create the service, as well as the Helm chart of the service.
-For this purpose we need the helm charts as a tar.gz archive. To archive it use following command:
+After creating the project, we can continue by onboarding the *helloservice* as a service to your project using the `keptn create service` and `keptn add-resource` commands. You need to pass the project where you want to create the service, as well as the Helm chart of the service.
+For this purpose we need the helm charts as a tar.gz archive. Our repository already contains a tar.gz archive of our helm chart but normally you could use the following command for the conversion:
 
 <!-- command -->
 ```
-tar cfvz ./helm-charts/helloserver.tgz ./helm-charts/helloserver
+tar cfvz ./helm/helloservice.tgz ./helm/helloservice
```

Comment:
Either we remove the .tgz file from https://github.com/keptn/examples/tree/master/quickstart/demo/helm or we remove the step from this tutorial. But having both is redundant and might confuse the user.

In order to be in-line with the other tutorials, we should remove the step from the tutorial.

Reply:
That is fine. I will remove it in the next commit.